spec/models/list_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
```

Comment:
Shouldn't you test something here? Like you did in the specs for the other models.